Components/Windows/create-local-user/README.md

```diff
@@ -1,8 +1,8 @@
 # Creation of a local Windows user account with RDP login permissions
 
-This is a sample component that demonstrates how to create a local Windows user account without Administrator permissions and grant RDP login privileges to this account. This can be useful in situation where you build a jump host that you do not want to join into an Active Directory and where you do not want the users to have admin access, especially if you manage your configuration changes to the machines via an EC2 Image Builder pipeline.
+This is a sample component that demonstrates how to create a local Windows user account without Administrator permissions and grant RDP login privileges to this account. This can be useful in a situation where you build a jump host that you do not want to join into an Active Directory domain and where you do not want the users to have admin access, especially if you manage your configuration changes to the machines via an EC2 Image Builder pipeline.
 
-This component activates the AWS Tools for Powershell for Windows [documentation](https://docs.aws.amazon.com/powershell/latest/userguide/pstools-welcome.html) which are already pre-installed on the AWS Managed AMIs for Windows Server 2019 and probably other versions as well. This is done so that the Powershell part of this component can interact with AWS Secrets Manager to retrieve the password for this users, as we did not want to have it written down in the document. To allow the build instance to retrieve the password for the user from AWS Secrets Manager the IAM role assigned to it via the EC2 Image Builder Infrastructure configuration needs to have a policy attached which allows the action ```secretsmanager:GetSecretValue``` for the specified secret.
+This component activates the AWS Tools for Powershell for Windows [documentation](https://docs.aws.amazon.com/powershell/latest/userguide/pstools-welcome.html) which are which are pre-installed on all AWS Windows AMIs. This is done so that the Powershell part of this component can interact with AWS Secrets Manager to retrieve the password for this users, as we did not want to have it written down in the document. To allow the build instance to retrieve the password for the user from AWS Secrets Manager the IAM role assigned to it via the EC2 Image Builder Infrastructure configuration needs to have a policy attached which allows the action ```secretsmanager:GetSecretValue``` for the specified secret.
```

Comment:
It looks like this duplicates `which are`.

```... which are which are pre-installed ...```

Reply:
You are right. I removed the duplicate.